<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>376</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>376</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item row="4" column="0">
      <widget class="QPushButton" name="button_7">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button_8">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="button_5">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="button_minus">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="button_3">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="button_ce">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="button_4">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="button_plus">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="button_multiply">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>ร</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="button_divide">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>รท</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="button_power">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="button_equal">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #608BCC;</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="button_sqrt">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>โ</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="button_2">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="button_0">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="button_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2" colspan="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Andale Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="button_6">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="button_dot">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="minimumSize">
        <size>
         <width>358</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>356</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Andale Mono</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>BlankCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="button_sign">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #fff;
background-color: #AAB3B6;</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_9">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>32</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #ECECEC;</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
